packages/genesys-spark-components/src/components/stable/gux-listbox/gux-listbox.service.ts

```diff
@@ -66,11 +66,12 @@ function setActiveOption(
 }
 
 function getFirstOption(list: HTMLGuxListboxElement): ListboxOptionElement {
+  console.log(getAvailableListOptions(list)[0]);
```

Comment:
remove

Reply:
Good spot, removed across this and parent branch